scripts/tests/chiptest/test_definition.py

```diff
@@ -454,40 +456,51 @@ def Run(self, runner, apps_register, paths: ApplicationPaths, pics_file: str,
                 setupCode = app.setupCode
 
             if test_runtime == TestRunTime.MATTER_REPL_PYTHON:
-                python_cmd = (paths.matter_repl_yaml_tester_cmd
-                              + ['--setup-code', setupCode]
-                              + ['--yaml-path', self.run_name]
-                              + ["--pics-file", pics_file])
+                python_cmd = apps.matter_repl_yaml_tester_cmd \
+                    .add_args(('--setup-code', setupCode)) \
+                    .add_args(('--yaml-path', self.run_name)) \
+                    .add_args(("--pics-file", pics_file))
+
                 if dry_run:
-                    logging.info(" ".join(python_cmd))
+                    logging.info("{} {}".format(python_cmd.path, " ".join(python_cmd.args)))
```

Comment:
What if there is some wrapper used? Why not `.to_cmd()` since you have such API?